app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+
+  def create
+    comment = Comment.create(comment_params)
+    job = Job.find(comment.job_id)
+    company = Company.find(job.company_id)
+    redirect_to company_job_path(company, job)
```

Comment:
since we already have the job, we don't really need the company...we could do `redirect_to company_job_path(job.company, job)`